### DISCUSSION TITLE
Open drain output

### DISCUSSION THREAD

**Discussion by SgtSiff** [AxxAxx/AxxSolder#36] [Ideas]
Hiya, love this project and anxiously waiting for Rev3.

I wondered if it would be possible to integrate a single open drain output?

My thinking is this could be used to trigger or drive an extractor fan when the iron was active.

Edit: I realise space on the board is at a premium - something like capable of sinking <=1A at 24V would be adequate.

What do you think?

**Comment by AxxAxx** [comment]
Good idea to have some signal to external equipment. 
But indeed space is sort of limited. It also increases the BOM even more.

It could make more sense to have a dedicated GPIO for external equipment.
I could add a header for "external tools" with a GND and 3.3V logic output. 
That way anyone could hook up any relay module of mosfet switch to drive whatever fan or equipment. 

There should also be a software feature in the settings to drive this logic output high whenever AxxSolder is in "RUN" state. 

Would do you think?

**Reply by SgtSiff** [reply]
I think that's a great idea, just giving a GPIO as opposed to an open drain gives great flexibility whilst not changing the bom price.

Thanks for considering this idea
